packages/zoe/test/unitTests/test-instanceStorage.js

```diff
@@ -33,71 +30,6 @@ const setupIssuersForTest = () => {
   return { currencyKit, ticketKit };
 };
 
-test('makeAndStoreInstanceRecord', async t => {
```

Comment:
Was this test moved? No longer relevant?

Reply:
Dropped because it was testing a separate function that had to be moved in-line. The functions returned on lines 51-57 are now methods on the instanceRecord object.

That's another place where the naming is confused. The InstanceRecord object has a `getInstanceRecord()` method.  so InstanceRecord refers both to this durable kind and to the data it holds.

`InstanceRecordStorage` serves two purposes. On the Zoe side, there will be many contract instances, and each has an InstanceRecord holding terms, issuers, brand, installation, etc.  On the ZCF side, there will only be the one instance, but the same object is used to manage the data.